addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/type/generator/CcuMetadataExtractor.java

```diff
@@ -52,8 +52,8 @@ public static void main(String[] args) {
             CcuMetadataExtractor dg = new CcuMetadataExtractor();
             dg.generate();
 
-        } catch (Exception ex) {
-            ex.printStackTrace();
+        } catch (IOException ex) {
+            System.err.println(ex.getMessage());
```

Comment:
FTR: This is inside a static void main, if another exception will be throw it will still log a stacktrace when exiting only in case an ioexception the behaviour changes, but this follows our guidelines. 

@gerrieg your approval is appreciated 🙂